client/src/Components/Pages/Dashboard/ActiveJobs/index.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import Input from '../../../CommonComponents/Inputs';
+import Button from '../../../CommonComponents/Button';
+
+import "./style.css";
+
+export default function index(props){
+  const { action } = props;
+
+return(
+  <React.Fragment>
+  <div className='find'>
+  <h2>Find Job</h2>
+<Input  type={'text'} label={'Post code'} name={'post-code'} width={'150px'} placeholder={'post code'} />
+
+<label>Subject  :
+   <select>
+      <option selected>Marie Clare</option>
+      <option >Ahmad</option>
+      <option >Marwan</option>
+   </select> </label>
+
+   <Button className='button' value={'Search'} color={'#5D6A8B'} onClick={action}/>
```

Comment:
In a future PR, when you're ready to wire things up, I think it would be good if the post code and selected subject were passed as arguments to the action() function.

Reply:
Yes, of course, I plan for that, but in that phase we where work in the static parts, the future updates will depend on the requirements of the functionality.